tests/20-rdfc-verify.js

```diff
@@ -1,84 +1,183 @@
 /*!
- * Copyright 2023 Digital Bazaar, Inc.
+ * Copyright 2024 Digital Bazaar, Inc.
  * SPDX-License-Identifier: BSD-3-Clause
  */
-import {verificationFail, verificationSuccess} from './assertions.js';
-import {endpoints} from 'vc-test-suite-implementations';
+
+import {
+  verificationFail, verificationSuccess
+} from './assertions.js';
+import {
+  endpoints as registeredImplementations
+} from 'vc-test-suite-implementations';
+
 import {getSuiteConfig} from './test-config.js';
+
+import {expect} from 'chai';
 import {issueTestData} from './vc-generator/index.js';
 import {klona} from 'klona';
 
+/**
+ * Builds a result cell for the test reporter with the expected parameters.
+ *
+ * This is used to annotate the test results with the implementation name and
+ * test name.
+ *
+ * @param {object} object - The object to use for building the result cell.
+ * @param {string} object.name - The name of the implementation.
+ * @param {string} object.keyType - The `keyType` being tested (e.g., `P-256`)
+ * @param {string} object.testTitle - The title of the test.
+ * @returns {object} The result cell object.
+ */
+const buildResultCell = ({name, keyType, testTitle}) => ({
```

Comment:
this is should definitely be in a helper file. I actually think this would work in just `helpers.js` as we obviously will want this in other tests that uses keyType.